### ISSUE TITLE
Excludes serialization incorrect in JSON and YAML examples

### ISSUE THREAD

**Issue by tsteenbe** [spdx/spdx-spec#401]
Examples contain `packageVerificationCodeExcludedFiles` that do not make sense for people familiar with JSON and YAML.

```
packageVerificationCodeExcludedFiles:
      - "excludes: ./package.spdx"
```
should be 
```
packageVerificationCodeExcludedFiles:
      - "./package.spdx"
```
see https://github.com/spdx/spdx-spec/blob/development/v2.2.1/examples/SPDXYAMLExample-2.2.spdx.yaml#L231

Same issue in JSON `"packageVerificationCodeExcludedFiles" : [ "excludes: ./package.spdx" ],` should simple be `"packageVerificationCodeExcludedFiles" : [ " ./package.spdx" ],` see example https://github.com/spdx/spdx-spec/blob/development/v2.2.1/examples/SPDXJSONExample-v2.2.spdx.json#L102

**Comment by goneall** [comment]
Completely agree.  I would call this a bug.  I'll try to update the examples and fix the Java tools in the next week or so.

**Comment by goneall** [comment]
Fixed in #416
